perun-wui-core/src/main/java/cz/metacentrum/perun/wui/client/utils/UiUtils.java

```diff
@@ -136,6 +144,28 @@ public void onClick(ClickEvent event) {
 	}
 
 	/**
+	 * add dropdown to set
+	 *
+	 * @param table        Table to bind filtering to
+	 */
+	public static <T extends JavaScriptObject,  C extends SuggestBox> void bindDropdown(final HashMap<AnchorListItem, PerunColumnType> anchorMap, final PerunDataGrid<T> table){
+		Set<AnchorListItem> keySet = anchorMap.keySet();
+		for (final AnchorListItem anchor : keySet){
+			anchor.addClickHandler(new ClickHandler() {
+				@Override
+				public void onClick(ClickEvent event) {
+					if (anchor.getIcon().equals(IconType.CHECK)){
```

Comment:
Please switch equals params. It's safer to call it on enum which can't be null, rather than passed param, which can be null. 
```java
IconType.CHECK.equals(anchor.getIcon())
```
Also please drop the minus icon from filter widget (each ui.xml). We will switch ``IconType.CHECK`` and ``null`` value. Which means, there is no need to define icon in ui.xml.

Please add this call before adding ClickHandler to the Anchor.
```java
anchor.setIconFixedWidth(true);
```
It will ensure, that text is properly aligned even if there is no icon set. Thank you.